### PR TITLE
Fix broken release notes links

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
**Description**

Changes proposed in this pull request:

- Release notes links point to the specific version on the website which is no longer supported
- This PR replaces the website links with GitHub links

**Related issue(s)**

### CODE REVIEW

**Review by majakurcius — DISMISSED** [review]

File: content/blog-posts/2021-03-16-release-notes-1.20/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Starting with the 1.20 release, you can install or upgrade Kyma with one of thes
 - **Evaluation profile** with limited resources, suitable if you want to try out Kyma
 - **Production profile** with high availability and scalability, recommended if you use Kyma for your production workload
 
-For more details, read the [overview of both profiles](https://kyma-project.io/docs/1.20/root/kyma/#installation-overview-profiles).
+For more details, read the [overview of both profiles](https://github.com/kyma-project/kyma/blob/main/docs/kyma/04-01-overview.md#profiles).
```
Comment (klaudiagrz) [inline content/blog-posts/2021-03-16-release-notes-1.20/index.md:41]:
```suggestion
For more details, read the [overview of both profiles](https://github.com/kyma-project/kyma/blob/1.20.0/docs/kyma/04-01-overview.md#profiles).
```

Comment (klaudiagrz) [reply content/blog-posts/2021-03-16-release-notes-1.20/index.md:41]: Please change the links do that they point to the specific version instead of `main`

**Review by klaudiagrz — CHANGES_REQUESTED** [review]
Please change links to the specific gh version

**Review by klaudiagrz — APPROVED** [review]
👍

**Pull request merged**